start-gateway-config.sh

```diff
@@ -19,6 +19,13 @@ echo 153 > /sys/kernel/debug/bluetooth/hci0/adv_max_interval
 # Disable pairing
 printf "pairable off\nquit" | /usr/bin/bluetoothctl
 
-# Check dbus container is ready and then launch config
-wait_for_dbus \
-	&& python gatewayconfig
+
+prevent_start="${PREVENT_START_GATEWAYCONFIG:-0}"
+if [ "$prevent_start" = 1 ]; then
```

Comment:
We had another pattern for this but I can't find it. In gwmfr?

Reply:
https://github.com/NebraLtd/hm-gwmfr/blob/1abdf511c71a59a8c24b411f32a7b2117e17b5da/eccProg.py#L72-L78

Reply:
That's surprising. I confirmed that you can still stop the container from Balena console using this approach.

Reply:
@marvinmarnold what I meant was, it was designed to stop the container from exiting after the code finished the key programming. Rather than to stop the key programming from starting.

Whereas here it looks like you are trying to stop it starting.

Hope that makes more sense

Reply:
Ah ok. My thinking for adding this before app startup is that sometimes our Python code breaks and causes crash restart loops. Having this feature would allow us to SSH into the container and debug without having to redeploy.

Reply:
Yeah that makes more sense for sure 👍